index.src.html

```diff
@@ -610,6 +610,11 @@ <h2 id="mitigations">
     Mitigation Strategies
   </h2>
 
+  To mitigate the security and privacy risks you’ve identified in your
+  specification as you’ve filled out the questionnaire and consulted with PING,
```

Comment:
This is possibly not a good idea, since we're again suggesting a soft requirement-like for consulting with PING (where in case of a TAG review, for example, we refrained from that. Let's simplify possibly?

Reply:
Sure.